setup.py

```diff
@@ -12,16 +12,16 @@
     name = 'django-datadog',
     version = '0.1.0',
     packages = [
-        'datadog',
-        'datadog.middleware'
+        'django-datadog',
+        'django-datadog.middleware'
```

Comment:
Again: Please don't use a dash in module and package names, unless you have a _very_  good reason for it.